gatsby-config.js

```diff
@@ -240,3 +229,25 @@ module.exports = {
     FAST_DEV: true, // DEV_SSR, QUERY_ON_DEMAND & LAZY_IMAGES
   },
 }
+
+// Avoid loading Matomo in Gatsby Cloud envs since NODE_ENV is `production` in
+// there and it will send testing data as production otherwise
+if (!isGatsbyCloud) {
+  config.plugins = [
+    ...config.plugins,
+    // Matomo analtyics
+    {
+      resolve: "gatsby-plugin-matomo",
+      options: {
```

Comment:
The Matomo plugin also has a `dev` option:
https://github.com/kremalicious/gatsby-plugin-matomo#options

Should we set that to `true` when in dev env @pettinarip?